src/ExtensionsSample/Models/ItemDoc.cs

```diff
@@ -10,6 +10,9 @@ public class ItemDoc
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        [JsonProperty("partition")]
+        public string Partition { get; set; }
```

Comment:
I must have backed that out. Fixed.

---
In reply to: [59794188](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/63#discussion_r59794188) [](ancestors = 59794188)